app/src/main/java/org/fossasia/openevent/general/search/SearchFilterFragment.kt

```diff
@@ -33,6 +34,7 @@ class SearchFilterFragment : Fragment() {
     private val searchViewModel by viewModel<SearchViewModel>()
     private val safeArgs: SearchFilterFragmentArgs by navArgs()
     private lateinit var sortBy: String
+    private var isSessionsAndSpeakersChecked = false
```

Comment:
Normally, I would put `isSessionsAndSpeakersChecked` inside ViewModel instead of Fragment, just to be sure that it survives screen rotation

Reply:
Okay got it. I will also move other Variables inside view models.